scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -442,9 +442,11 @@
   (timers/time! (metrics/timer "controller-process" name)
     (loop [{:keys [cook-expected-state waiting-metric-timer] :as cook-expected-state-dict} (get @cook-expected-state-map pod-name)
            {:keys [synthesized-state pod] :as k8s-actual-state-dict} (get @k8s-actual-state-map pod-name)]
-      (log/info "In" name "compute cluster, processing pod" pod-name ";"
-                "cook-expected:" (prepare-cook-expected-state-dict-for-logging cook-expected-state-dict) ","
-                "k8s-actual:" (prepare-k8s-actual-state-dict-for-logging k8s-actual-state-dict))
+      ; Skip logging when both Cook and Kubernetes think the pod is running
```

Comment:
This trims the log too much. We want to track running-running EXCEPT when doing a scan. So this should be conditioned on a a third thing being true, `doing-scan?`.

Reply:
For my knowledge, what are the types of things we would learn from this log in a running-running state? Yes, we would have a log line showing that the instance did hit the running state, but what other information can we get?

I can certainly plumb `doing-scan?` through, but I need clarification on how the accesses differ.

Reply:
The part of the log we're trying to trim is that every 2 minutes, we scan everything, and print out all of the running-running states (repeatedly). Those print lines are redundant because they show an unchanged state.

However, any visit of the running-running state NOT as part of that scan is triggered on a state change. If we trim those, we also trim some state changes. I.e., we're losing information compared to before. In my analysis, I scoped log diet to only removing things of zero or near zero value.

Reply:
Ahhh - makes sense. Ok. Thanks for the context.